.github/workflows/ignore_symlinks.yml

```
@@ -0,0 +1,72 @@
+        fi
+
+        # Verify that ignored path is not included
+        notexpect="testfiles/ignore/symlink.sh"
+        if [[ "${{ steps.check.outputs.files }}" =~ $notexpect ]];then
+          echo "::error:: Unexpected file $notexpect found in ${{ steps.check.outputs.files }}"
+          exit 1
+        fi
+
+    - name: Run ShellCheck (include symbolic links)
+      uses: ./
+      id: check
```

actionlint

step ID "check" duplicates. previously defined at line:28,col:11. step ID must be unique within a job. note that step ID is case insensitive

(id)